js/login.js

```diff
@@ -0,0 +1,93 @@
+(function () {
+  const emailInput = document.querySelector("#login--id");
+  const emailError = document.querySelector(".form__email-error");
+
+  const passwordInput = document.querySelector("#login--password");
+  const passwordError = document.querySelector(".form__password-error");
+
+  const hidePassword = document.querySelector(".form__password-hidden");
+
+  const loginBtn = document.querySelector(".form__button");
+
+  // 이메일 유효성 검사
+  function isValidEmail(email) {
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+    return emailPattern.test(email); // true or false 반환
+  }
+
+  // 이메일 체크
+  emailInput.addEventListener("focusout", () => {
+    const email = emailInput.value.trim();
+
+    if (email === "") {
+      emailError.textContent = "이메일을 입력해주세요.";
+      emailError.parentElement.classList.add("form-error");
+    } else if (!isValidEmail(email)) {
+      emailError.textContent = "잘못된 이메일 형식입니다.";
+      emailError.parentElement.classList.add("form-error");
+    } else {
+      emailError.textContent = "";
+      emailError.parentElement.classList.remove("form-error");
+    }
+  });
+
+
+  //비밀번호 체크
+  passwordInput.addEventListener("focusout", () => {
+    const password = passwordInput.value.trim();
+    if (password === "") {
+      passwordError.textContent = "비밀번호를 입력해주세요.";
+      passwordError.parentElement.classList.add("form-error");
+      return false;
```

Comment:
이벤트 리스너라서 따로 값을 리턴하지 않아도 될 거 같아요!
```suggestion
      return
```